index.html

```diff
@@ -7,14 +7,20 @@
 	<title>Portfolio</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="styles.css">
+
 	
 </head>
 
 <body>
 	<header class="container">
 		<div id="logo">Lassana</div>
-		<nav>
+		<nav class="nae toggle-menu">
 			<button id="hide" type="button"><i class="fas fa-bars"></i></button>
+			<ul class="mobile-menu" id="anchors">
+				<li class="pad"><a href="#works">Portfolio</a></li>
+				<li  class="pad"><a href="#about-myself">About</a></li>
+				<li  class="pad"><a href="#contact">Contact</a></li>
```

Comment:
As of this [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#project-requirements) when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Currently, it's not taking the user to the correct part of the page. Kindly, update the `href` with the appropriate `id` names.